src/main/java/com/nina/modelling/shoes/WomanShoe.java

```diff
@@ -0,0 +1,27 @@
+package com.nina.modelling.shoes;
+
+public class WomanShoe extends Shoe {
+    private String collections;  //TODO: probably list of strings
```

Comment:
probably collections is an list of strings?
maybe accessories also if a shoe has more accessories

Reply:
Named the collections to group, it's synonymy.